### PR TITLE
Simple board with move selection.

### PR DESCRIPTION
**Pull request by shaper**

### CODE REVIEW

**Comment by shaper** [comment]
We can review further when I get to adding some tests, will land for now.

File: words/source/MovePicker.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    _willReceiveProps(props, state) {
+      if (!state.dictionaryLoadingStarted) {
+        let particleRef = this;
+        // TODO(wkorman): Replace this with our own hosted dictionary.
+        const dictionaryName =
+          'https://raw.githubusercontent.com/shaper/shaper.github.io/master/resources/american-english.txt';
+        info(`Loading dictionary [name=${dictionaryName}].`);
+        fetch(dictionaryName).then(response =>
+          response.text().then(text => {
+            let dictionary = new Dictionary(text);
```
Comment (smalls) [inline words/source/MovePicker.js]: Not stored in state yet because it isn't being used?

File: words/source/MovePicker.js [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+        }
+      }
+      state.move.coordinates = newCoordinates;
+      this._setState({
+        board: state.board,
+        move: state.move,
+        selectedTiles: state.selectedTiles
+      });
+    }
+    _onSubmitMove(e, state) {
```
Comment (smalls) [inline words/source/MovePicker.js]: I wonder if there's a more descriptive term for 'move'. 'Turn', or 'selected'?

**Comment by smalls** [comment]
I'm glad you merged it, don't wait for me. Some small questions interleaved.

File: words/Board.schema [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Copyright (c) 2017 Google Inc. All rights reserved.
+# This code may only be used under the BSD style license found at
+# http://polymer.github.io/LICENSE.txt
+# Code distributed by Google as part of this project is also
+# subject to an additional IP rights grant found at
+# http://polymer.github.io/PATENTS.txt
+
+schema Board
+  normative
+    # A string list of letters comprising the board laid out in an
```
[the remark this answers is not in the document]
Comment (shaper) [reply words/Board.schema]: Good call, will do, though I also hope to change this to an ordered list of Number tuples soon.

File: words/BoardGenerator.manifest [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# @license
```
[the remark this answers is not in the document]
Comment (shaper) [reply words/BoardGenerator.manifest]: I definitely found the different .recipes, .schema, .manifest suffixes confusing initially. Currently am following naming approach used by most all of the other demos, open to changing, I suspect these things are somewhat amorphous over time. Let's discuss in person at some point.

File: words/index.test.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  <meta charset="utf-8">
+  <title>Words Mocha Tests</title>
+  <link href="./node_modules/mocha/mocha.css" rel="stylesheet" />
+</head>
+<div id="mocha"></div>
+<div id="target"></div>
+<script src="./node_modules/chai/chai.js"></script>
+<script src="./node_modules/mocha/mocha.js"></script>
+<script>mocha.setup('bdd')</script>
+<!-- TODO(wkorman): Fancy tests go here. -->
```
[the remark this answers is not in the document]
Comment (shaper) [reply words/index.test.html]: I've added some now! Not so fancy though. https://www.youtube.com/watch?v=C9VYU0YZSbo

File: words/source/MovePicker.js (hunk 1 above)
Comment (shaper) [reply words/source/MovePicker.js]: Yes, it's being stored and used now, though, excitingly.

File: words/source/MovePicker.js (hunk 2 above)
Comment (shaper) [reply words/source/MovePicker.js]: I like 'turn', and agree that 'move' isn't a great word. Will look at revising soon.

**Pull request merged**